app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    [Person(person["name"], person["age"]) for person in people]
```

Comment:
The list comprehension here creates `Person` instances but does not store them in a variable or return them. Since the instances are stored in the class-level dictionary, this line is effectively just iterating over `people` and can be simplified to a regular for loop if the side effect of storing in `Person.people` is intended.